srv/salt/state/dask_in_python39_with_pip.sls

```diff
@@ -0,0 +1,9 @@
+# File:/srv/salt/dask_in_python39_with_pip.sls
+
+install_dask_in_python39_with_pip:
+  pip.installed:
+    - names:
+      - dask
+    - bin_env: '/usr/bin/pip3.9'
+    - require:
+      - install_python39  # installs python39 and python39-pip
```

Comment:
this is an independent salt state from the conda install dask I assume? otherwise its redundant and I do believe we can install dask with conda using pip.

Reply:
Yes. This is independent.